tests/qos/test_qos_sai.py

```diff
@@ -1559,6 +1570,12 @@ def testQosSaiQSharedWatermark(
                 qosConfig = dutQosConfig["param"][portSpeedCableLength]
             triggerDrop = qosConfig[queueProfile]["pkts_num_trig_ingr_drp"]
         else:
+            if not get_src_dst_asic_and_duts['single_asic_test'] and \
```

Comment:
the fixture get_src_dst_asic_and_duts does not hold the value 'single_asic_test'. If the check is required to verify single_asic  test then below can be added 
 src_dut_index == dst_dut_index && src_asic_index == dst_asic_index

Reply:
@ansrajpu-git : I added this key in the same PR.